kura/org.eclipse.kura.wire.provider/src/main/java/org/eclipse/kura/internal/wire/WireServiceImpl.java

```diff
@@ -206,13 +209,13 @@ synchronized void createWires() throws KuraException {
                                 conf.setWire(wire);
                             }
                         } catch (final InvalidSyntaxException e) {
-                            s_logger.error(ThrowableUtil.stackTraceAsString(e));
+                            logger.error(ThrowableUtil.stackTraceAsString(e));
```

Comment:
Same here as the passed arguments to exceptions. The logger can handle stack traces by itself and there is not need to pre-render this.